src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <React.Fragment key={movie.imdbId}>
+        <MovieCard movie={movie} />
+      </React.Fragment>
```

Comment:
Remove unnecessary Fragment
```suggestion
        <MovieCard movie={movie} key={movie.imdbId} />
```